finality-provider/service/fp_instance.go

```diff
@@ -1020,3 +1026,8 @@ func (fp *FinalityProviderInstance) GetFinalityProviderSlashedOrJailedWithRetry(
 
 	return slashed, jailed, nil
 }
+
+// TestGetPubProof only used for tests to get access to the store
+func (fp *FinalityProviderInstance) TestGetPubProof(height uint64) ([]byte, error) {
```

Comment:
This is probably not needed as we can access the store by `app.GetPubRandProofStore()`